src/constructs/iam/policies/describe-ec2.test.ts

```diff
@@ -0,0 +1,38 @@
+import "@aws-cdk/assert/jest";
+import { Role, ServicePrincipal } from "@aws-cdk/aws-iam";
+import { simpleGuStackForTesting } from "../../../../test/utils";
+import { GuDescribeEC2Policy } from "./describe-ec2";
+
+describe("DescribeEC2Policy", () => {
+  it("can accept a custom policy name", () => {
+    const stack = simpleGuStackForTesting();
+
+    const policy = new GuDescribeEC2Policy(stack, "DescribeEC2Policy", { policyName: "my-awesome-policy" });
+
+    // IAM Policies need to be attached to a role, group or user to be created in a stack
+    policy.attachToRole(
+      new Role(stack, "TestRole", {
+        assumedBy: new ServicePrincipal("ec2.amazonaws.com"),
+      })
+    );
```

Comment:
This could use the new `attachPolicyToTestRole` function

Reply:
Oh yeah!

Reply:
Updated.